docs/_static/rtd_theme_overrides.css

```diff
@@ -71,14 +103,206 @@ div.pkgnav ul li {
 }
 
 div.pkgnav ul li a {
-    margin: 0;
-    /*padding: 0 5px 0 5px;*/
-    /*line-height: 1.75em;*/
     color: #f9f9f0;
+    margin: 0;
     text-shadow: 0 0 1px rgba(0, 0, 0, 0.5);
+    /*line-height: 1.75em;*/
+    /*padding: 0 5px 0 5px;*/
 }
 
 div.pkgnav ul li a:hover {
-    color: white;
+    color: #f50404;
     text-shadow: 0 0 1px rgba(255, 255, 255, 0.5);
 }
+
+/* Styles for text in the footer chosen to improve contrast. */
+
+footer p {
+    color: #222222;
+}
+
+/*
+  Styles for hyperlinks. The colors must meet the WCAG AA standard
+  for contrast. Link colors can be tested for accessibility at:
+  https://webaim.org/resources/linkcontrastchecker
+*/
+
+div.wy-nav-content a:link { /* still changing links in sidebar!!!! */
```

Comment:
- [ ] Make sure this comment is addressed in more recent changes I made, and if so, then remove it